common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -5970,16 +5971,27 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
-  val CLIENT_TAGS_EXPR: ConfigEntry[String] =
-    buildConf("celeborn.client.tagsExpr")
-      .categories("client")
+  val TAGS_EXPR: ConfigEntry[String] =
+    buildConf("celeborn.tags.tagsExpr")
+      .categories("tags", "master", "client")
```

Comment:
There is no such thing as called "tag" in celeborn config items. The category is a component, for example, there is "client".